Jenkinsfile

```diff
@@ -94,7 +94,32 @@ pipeline {
                     '''
                 }
             }
-        }  
+        }
+
+        stage('Run Turbine Unit Test Suite') {
+            options {
+                timeout(time: 30, unit: 'MINUTES') 
+            }
+                steps {
+                    withEnv(["PATH=$PATH:~/.local/bin;NOBUILD=true"]) {
+                        withDockerRegistry([url: 'https://index.docker.io/v1/', credentialsId: 'docker.com-bot']) {
+                            sh '''#!/usr/bin/env bash
+
+                            # Run eslint on turbine code
+                            cd src/pfe/file-watcher
+                            npm install
+                            npm run eslint
+                            if [ $? -ne 0 ]; then
+                                exit 1
+                            fi
+                            
+                            # Run the unit test suite
+                            echo "Started running Turbine Unit Test Suite"
+                            npm run unit:test
+                            '''
+                        }
+                }
+        }
```

Comment:
I think you are missing an end `}` which caused the build to fail https://ci.eclipse.org/codewind/job/Codewind/job/codewind/job/PR-554/2/console

Reply:
Yeah, I added that but doesn't look like my change is getting picked up.

Reply:
Nevermind, there were 2 missing brackets. Build looks to be getting further now.

Reply:
Yeah it was missing two brackets...haha looks like I missed two brackets on my PR as well and you copied it from mine